shift_detector/checks/lda_check.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+from shift_detector.checks.check import Check, Report
+from shift_detector.precalculations.lda_embedding import LdaEmbedding
+from shift_detector.utils.column_management import ColumnType
+from collections import Counter
+
+
+class LdaCheck(Check):
+
+    def __init__(self, significance=10, n_topics=20, n_iter=10, lib='sklearn', random_state=0,
```

Comment:
Please add comments to explain the parameters. For the statistical checks, significance is always between 0 and 1, because it is a probability. I find it a little confusing to see values larger than 1 for significance. What is this value used for?